.all-contributorsrc

```diff
@@ -1231,3 +1231,11 @@
   "repoHost": "https://github.com",
   "skipCi": true
 }
+"login": "aditi2906",
+      "name": "Aditi Tripathi",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/61978541?v=4",
+      "profile": "https://github.com/aditi2906",
+      "contributions": [
+        "code",
+        "content"}
+        
```

Comment:
No need to add this.
Our bot will handle this once your PR gets merged into the codebase.